pkg/virt-controller/watch/pool/pool.go

```diff
@@ -527,22 +527,14 @@ func filterVMs(vms []*virtv1.VirtualMachine, f func(vmi *virtv1.VirtualMachine)
 	return filtered
 }
 
-func resolveBasePolicy(scaleInStrategy *poolv1.VirtualMachinePoolScaleInStrategy) poolv1.VirtualMachinePoolBasePolicy {
-	if scaleInStrategy == nil {
+func resolveScaleInPolicy(scaleInStrategy *poolv1.VirtualMachinePoolScaleInStrategy) poolv1.VirtualMachinePoolBasePolicy {
+	if scaleInStrategy == nil || scaleInStrategy.Proactive == nil {
```

Comment:
Please push into the right commit. The commit title is `Add E2E tests for VMpool update strategy`.